src/components/GraphCard/StackedGraph.tsx

```diff
@@ -29,10 +30,11 @@ interface IGraphGroupMembers {
 }
 
 interface IStackedGraphProps {
-    data: IGraphGroupMembers[]
+    dataUsers?: IGraphGroupMembers[]
+    dataUserCategories?: IGroupMemberExpensesByCategoryDailyResponse[]
```

Comment:
you can write something like
```
interface IGraphProps = (
    | { dataUsers: IGraphGroupMembers[], dataUserCategories?: never }
    | { dataUsers:?: never, dataUserCategories: IExtendedUser[] }
)
```
to prevent developers from writing wrong props. check ChartCard for example